### DISCUSSION TITLE
Only 1 language shown

### DISCUSSION THREAD

**Discussion by janegle** [github-linguist/linguist#5293] [Classification] [answered]
In my repo https://github.com/janegle/EduFinderWebsite on the right hand side only HTML was shown. When I clicked on it, more languages are correctly detected and shown. How can I get them displayed in the homepage https://github.com/janegle/EduFinderWebsite on the right hand side?

**Comment by lildude** [comment, marked as answer]
🤔 your language analysis in the sidebar is consistent with the content of your `gh-pages` branch. I'm not sure why, but it could be the case if that was your default branch at the time and you've since changed. Try pushing a change to one of the files in the `src` directory on your `master` branch.
